src/app/static/src/app/components/Warning.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div v-if="warnings.length > 0">
         <h5 class="alert-heading pt-2">
-            <vue-feather type="alert-triangle" size="1.0x" class="custom-class mr-1 mb-1"></vue-feather>
+            <vue-feather type="alert-triangle" size="1.0" class="custom-class mr-1 mb-1"></vue-feather>
```

Comment:
This was causing an error in dev console

Reply:
Oops thanks yes, fixed this now by setting 1.0rem and also set the alignment on this. Now looks like 

![Screenshot_20231027_110156](https://github.com/mrc-ide/hint/assets/39248272/665d49ef-8794-4c5f-bf3c-5b45559be381)